src/pages/Home/Home.page.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import HomePage from './Home.page';
+
+describe('HomePage', () => {
+  it('renders', () => {
+    render(<HomePage />);
```

Comment:
all tests should have at least one assertion, otherwise they are not helping us 😄